data/net/minecraft/world/phys/Vec3.mapping

```diff
@@ -49,6 +55,9 @@ CLASS net/minecraft/world/phys/Vec3
 		ARG 1 axis
 	METHOD length ()D
 		COMMENT Returns the length of the vector.
+	METHOD lerp (Lnet/minecraft/world/phys/Vec3;D)Lnet/minecraft/world/phys/Vec3;
+		ARG 1 to
+		ARG 2 pct
```

Comment:
What on earth do these mean? What is `pct`?

Reply:
I named it `pct` as `Mth.lerp` has that parameter named `pct` as well. But yeah that name is not that meaningful. Probably means `percent` which is misleading as it's a number between 0 and one. I think I'll change the names in `Mth.lerp` as well then.